src/reader.rs

```diff
@@ -4495,6 +4511,7 @@ fn command_ends_paging(c: ReadlineCmd, focused_on_search_field: bool) -> bool {
         | rl::HistoryPager
         | rl::BackwardChar
         | rl::ForwardChar
+        | rl::ForwardCharPassive
```

Comment:
this is missing the backward variant

Reply:
Good catch.

Reply:
I'm going to update that match block to remove the `_` case so that that's a compilation error. `_` shouldn't be left in there for cases where new items need to be manually classified.